config/samples/backends/ceph_thin_provisioning/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
```

Comment:
As per my previous comment, you can remove `ceph.yaml`, and replace the content of `kustomization.yaml` with the follwing:

```
resources:
- ../ceph/

patches:
- target:
    kind: OpenStackControlPlane
  patch: |-
    - op: replace
      path: /spec/glance/template/customServiceConfig
      value: |
        [DEFAULT]
        enabled_backends = default_backend:rbd
        [glance_store]
        default_backend = default_backend
        [default_backend]
        rbd_store_ceph_conf = /etc/ceph/ceph.conf
        store_description = "RBD backend"
        rbd_store_pool = images
        rbd_store_user = openstack
        rbd_thin_provisioning = True
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

```

By doing this we can `kustomize build ceph_thin_provisioning` and get the right control plane CR.